sdk/face/ai-vision-face-rest/src/parameters.ts

```diff
@@ -7,20 +7,34 @@ import {
   RecognitionModel,
   FaceAttributeType,
   FindSimilarMatchMode,
+  CreateCollectionRequest,
+  UserDefinedFieldsForUpdate,
+  AddFaceFromUrlRequest,
+  FaceUserData,
+  UserDefinedFields,
   CreateLivenessSessionContent,
-  CreateLivenessWithVerifySessionContent,
+  CreateLivenessWithVerifySessionMultipartContent,
+  CreateLivenessWithVerifySessionJsonContent,
 } from "./models.js";
 
 export type GetOperationResultParameters = RequestParameters;
 
 export interface DetectFromUrlBodyParam {
-  body?: { url: string };
+  body: { url: string };
```

Comment:
this one is still using anonymous model?

Reply:
Yes. It remains the same.